ecommerce/api_test.py

```diff
@@ -719,3 +734,92 @@ def test_not_redeemable(self):
             assert pick_coupons(self.user) == []
         for coupon in Coupon.objects.all().exclude(id=self.not_auto_or_attached_coupon.id):
             _is_coupon_redeemable.assert_any_call(coupon, self.user)
+
+
+class PriceTests(MockedESTestCase):
+    """
+    Tests for calculating prices
+    """
+
+    def test_calculate_run_price_no_coupons(self):
+        """
+        If there are no coupons for this program the price should be what get_formatted_course_price returned
+        """
+        course_run, user = create_purchasable_course_run()
```

Comment:
every test case begins with a call to `create_purchasable_course_run`, with the only variance being that some cases discard the `user` return value. can you move this into `setUp` or `setUpTestData` and assign `course_run` and `user` to `self`?